### PR TITLE
CON-215: Added release notes

### PR DESCRIPTION
**Pull request by samuelraeburn**

### CODE REVIEW

**Comment by samuelraeburn** [comment]
Todo list

- [ ] Agree on numbering
- [ ] Update supported versions
- [ ] Update release notes in JavaScript Connector (complete this review first then copy across)
- [ ] Ensure that all of these issues are in release notes: https://jira.rti.com/issues/?jql=project%20%3D%20%22CON%22%20AND%20fixVersion%20%3D%20Hercules%20and%20resolution%20!%3D%20unresolved

@alexcamposruiz  For this review I have just copy-pasted the release notes from Jira.

File: docs/release_notes.rst [hunk 1]
```diff
@@ -34,3 +22,40 @@ runs on most Windows, Linux and macOS platforms.
...
+Version 2.0.0 (TODO Numebring)
+------------------------------
+
+What's New in 2.0.0
+~~~~~~~~~~~~~~~~~~~
+
+What's Fixed in 2.0.0
+~~~~~~~~~~~~~~~~~~~~~
+
+CON-163: Creating two instances of Connector resulted in a license error
```
Comment (adelleolson) [inline docs/release_notes.rst:34]: For "What's Fixed" - Jira #s should be at the end of the description, not in the title, like this:
[RTI Issue ID CON-163]

**Comment by adelleolson** [comment]
We also have to make sure all links to other documents are updated to use the new doc tree structure.

**Comment by samuelraeburn** [comment]
https://jira.rti.com/issues/?jql=project%20%3D%20CON%20AND%20fixVersion%20%3D%20Hercules%20AND%20%22Resolved%20Versions%22%20is%20EMPTY

File: docs/release_notes.rst [hunk 2]
```diff
@@ -34,3 +22,92 @@ runs on most Windows, Linux and macOS platforms.
 `main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
+Version 2.0.0 (TODO Numebring)
```
Comment (samuelraeburn) [inline docs/release_notes.rst:25]:
```suggestion
Version 1.1.0
```

File: docs/release_notes.rst [hunk 3]
```diff
@@ -34,3 +22,92 @@ runs on most Windows, Linux and macOS platforms.
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (samuelraeburn) [inline docs/release_notes.rst:26]:
```suggestion
~~~~~~~~~~~~~
```

File: docs/release_notes.rst [hunk 4]
```diff
@@ -34,3 +22,92 @@ runs on most Windows, Linux and macOS platforms.
...
+
+What's New in 2.0.0
```
Comment (samuelraeburn) [inline docs/release_notes.rst:28]:
```suggestion
What's New in 1.1.0
```

File: docs/release_notes.rst [hunk 5]
```diff
@@ -34,3 +22,92 @@ runs on most Windows, Linux and macOS platforms.
...
+This functionality is now available in the Python and JavaScript bindings.
+When a disposed sample is received, the key values can be accessed.
+The syntax for accessing these key values is the same as when the sample
+contains valid data (i.e., using type-specific getters, or obtaining the entire
+sample as an object). When the instance state is NOT_ALIVE_DISPOSED, only the
+key values in the sample should be accessed.
+
+[RTI Issue ID CON-188]
+
+What's Fixed in 2.0.0
```
Comment (samuelraeburn) [inline docs/release_notes.rst:56]:
```suggestion
What's Fixed in 1.1.0
```

**Comment by rkorte** [comment]
@samuelraeburn, some issues/questions:
Is there another Jira for general Connector doc updates (e.g., for updating all links to other Connext DDS docs, since we have changed all doc paths in Hercules)? Or, if there's not much going on for Connector in Hercules, I could just make all doc updates under CON-215? What do you think?

Are there other doc impacts related to CON-177, CON-188, CON-209 and anything else?

Are you ready to change version and release to 1.1.0 in conf.py? (I wasn't sure if I should change it yet)

LICENSE.pdf is giving me trouble. git thinks I modified it, and it won't let me "discard changes" (well, it does, but it still keeps the document in staging; I don't want it in staging, I don't want to commit it but I had to)

File: docs/release_notes.rst [hunk 6]
```diff
@@ -5,32 +5,109 @@
...
+
+The SampleInfo class in *Connector* has been extended to provide access to the
+sample state, view state, and instance state fields. These new fields work the
+same as the existing fields in the structure (in *Connector* for Python they are
+the keys to the dictionary, in *Connector* for JavaScript they are the keys to the
+JSON Object).
+
+
+Support for accessing the key values of disposed instances
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
```
Comment (samuelraeburn) [inline docs/release_notes.rst:43]:
```suggestion
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
```

**Pull request merged**